### DISCUSSION TITLE
Possible to launch executable without opening Whisky?

### DISCUSSION THREAD

**Discussion by cuga-rajal** [Whisky-App/Whisky#282] [Q&A] [answered]
If an executable is already installed in a bottle, and I just want to launch it, is there a way to launch it without opening Whisky? For example a desktop alias. 

Whisky 1.0 is working quite well under Ventura 13.5 (Windows 10) using GPK beta3.

**Comment by Crd5** [comment, marked as answer]
If you try to open the app, holding shift terminal will be opened. There, you can copy wine command and run it without Whisky

**Reply by Githsams** [reply]
Can you explain please ?

**Comment by cuga-rajal** [comment]
Thanks, that might be the best option. Launch it in a shell script
